test/resources/Hooks/useCopied/index.test.js

```diff
@@ -0,0 +1,55 @@
+/** @jest-environment jsdom */
+import React from 'react'
```

Comment:
@Jamchello tried out the Kent Dodds "use a test component" approach here, lmk what you think. I think it does show nicely how to use the hook

Reply:
yeah think so, especially because I think we can just use the countdown component, should be pretty straightforward